### DISCUSSION TITLE
Deployment Issue

### DISCUSSION THREAD

**Discussion by ESHAYAT102** [karthikmudunuri/eldoraui#18] [General]
### How Can I Deploy The Portfolio Template After Customizing It

I want to deploy the website with vercel. I tried to run `vercel` in the CLI, but it threw an error - `Error: Command "npm install" exited with 1`. I don't know what to do now. So, please help me out with this issue.

**Comment by karthikmudunuri** [comment]
can u share the ss of the error u are getting

**Reply by ESHAYAT102** [reply]
I tried to deploy the portfolio template, and I even customized it, but I got some errors.

1. I installed the zip file from https://github.com/karthikmudunuri/portfolio-template.
2. Exported it and opened it in VSCode.
3. Ran this command `cd portfolio-template` => `npm install` => Got this error - 
    'added 686 packages, and audited 687 packages in 4m

    180 packages are looking for funding
      run 'npm fund' for details

    6 vulnerabilities (1 low, 2 moderate, 3 high)

    To address all issues, run:
      npm audit fix

    Run 'npm audit' for details.' => `npm audit fix --force` => 
    Got this - 
    '# npm audit report

    next  >=13.4.0 <14.1.1
    Severity: high
    Next.js Server-Side Request Forgery in Server Actions - https://github.com/advisories/GHSA-fr5h-rqp8-mj6g
    fix available via `npm audit fix --force`
    Will install next@14.2.5, which is a breaking change
    node_modules/next

    1 high severity vulnerability

    To address all issues (including breaking changes), run:
     npm audit fix --force' => `npm audit fix --force` => 
     
     '
    added 1 package, removed 2 packages, changed 6 packages, and audited 696 packages in 1m

    178 packages are looking for funding
      run `npm fund` for details

    found 0 vulnerabilities'

    => did the customizations => `npm run build`. => Got this -
    
    '> portfolio-site@1.0.0 build
     '> NODE_ENV=production next build

    'NODE_ENV' is not recognized as an internal or external command,
    operable program or batch file.'
    
    
    I know it's too long but please help me out, i don't know what to do now.

**Reply by ESHAYAT102** [reply]
## The heck bro !!!

Any solutions ???

**Comment by karthikmudunuri** [comment]
ill look into this

**Reply by ESHAYAT102** [reply]
Ok 😌

Thanks Bro.

**Reply by karthikmudunuri** [reply]
try installing with `npm install --legacy-peer-deps`

**Reply by ESHAYAT102** [reply]
Nice!
It's Working I Guess.

**Reply by karthikmudunuri** [reply]
more updates are happened in this week check out the website once https://eldoraui.site/
